src/main/java/ca/xqz/tweetmouth/Tweet.java

```diff
@@ -1,15 +1,31 @@
 package ca.xqz.tweetmouth;
 
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
 import java.util.Arrays;
+import java.util.Date;
 import java.util.List;
+import java.util.Locale;
 import java.util.function.Function;
 import java.util.function.ToLongFunction;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
 import java.util.stream.Collectors;
 
 public class Tweet {
 
+    private final static DateFormat OUTPUT_DATE_FORMAT = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
+    private final static DateFormat INPUT_DATE_FORMAT = new SimpleDateFormat("EEE MMM dd HH:mm:ss ZZZZZ yyyy",
```

Comment:
I believe so, since the output format is the format ES takes to represent date and the input format is the format we're getting in the tweets. Would not expect these to change.